apps/login/src/app/(login)/authenticator/set/page.tsx

```diff
@@ -41,23 +41,19 @@ export default async function Page(props: { searchParams: Promise<Record<string
     ? await loadSessionById(sessionId, organization)
     : await loadSessionByLoginname(loginName, organization);
 
-  async function getAuthMethodsAndUser(
-    serviceUrl: string,
-
-    session?: Session,
-  ) {
+  async function getAuthMethodsAndUser(session?: Session) {
     const userId = session?.factors?.user?.id;
 
     if (!userId) {
       throw Error("Could not get user id from session");
     }
 
-    return listAuthenticationMethodTypes({ serviceConfig, userId,
-    }).then((methods) => {
+    return listAuthenticationMethodTypes({ serviceConfig, userId }).then((methods) => {
```

Comment:
I think it would make sense to use async await here instead of 3 nested returns

Reply:
sure. makes sense 👍